packages/utils/src/buildImageSizeProp.ts

```diff
@@ -1,6 +1,12 @@
-import { type BreakpointValue } from '@/types/tailwindTypes'
-
-import { breakpoints } from '@/constants/tailwindConstants'
+const breakpoints = {
+  sm: '640px',
+  md: '768px',
+  lg: '1024px',
+  xl: '1280px',
+  '2xl': '1536px',
+} as const
+
+type BreakpointValue = keyof typeof breakpoints
```

Comment:
Don't we want to move `tailwindConstants`?

Reply:
Sorry - Just read the PR comment. Maybe we can move it under `utils/constants` - just as we do for `utils/types` for now - Until we have a better understanding. At least, this way, we don't have duplicates.